src/protocol/relayer.md

```diff
@@ -0,0 +1,59 @@
+# Layer 1 Relayer/Bridge Protocol
+
+The Fuel relayer/bridge protocol is a set of rules that govern the interaction between the Fuel blockchain and the
+Layer 1 (L1) blockchain (e.g. Ethereum).  
+
+The Fuel blockchain can emit messages that will be processed by the smart contract on the L1 blockchain. The smart
+contract on the L1 can also emit events that will be processed by the Fuel blockchain.
+This is used to move any data between the L1 blockchain and the Fuel blockchain.
+
+## Fuel Message Outbox
+
+The message outbox is the set of messages sent to the L1 blockchain from the Fuel blockchain.
```

Comment:
I'd still like to see more technical description on how the message sending works. I.e. how can I verify that a message was actually sent on L1?

Reply:
I have intentionally not included a lot of detail on the "outbox" section, since this issue is specifically for the "inbox" stuff. There just wasn't any info on any of this stuff in the specs, so I wanted to set aside some space for the "outbox" stuff as well.

I'll create an issue for this: https://github.com/FuelLabs/fuel-specs/issues/575